src/ontology/mondo-edit.obo

```diff
@@ -573443,7 +573438,7 @@ id: MONDO:1011527
 name: hepatic lipidosis, domestic cat
 def: "Hepatic lipidosis that occurs in domestic cat." [MONDO:patterns/nonhuman_disease]
 xref: OMIA:000548-9685 {source="MONDO:equivalentTo"}
```

Comment:
`OMIA:000548-9685` is marked as "redundant" (https://omia.org/OMIA000548/9685/). Consider changing to "MONDO:equivalentObsolete".